app/containers/EntityList/ChartTimeline.js

```diff
@@ -52,19 +54,34 @@ const ChartWrapper = styled.div`
   position: relative;
 `;
 
+/* eslint react/prop-types: 0 */
+const HintWrapper = (props) => (
+  <Hint
+    {...props}
+    value={props.value}
+    marginTop={0}
+    marginLeft={0}
+  >
+    {props.children}
+  </Hint>
+);
+
 const myTimeFormat = (value) => {
   const formatted = timeFormat('%Y')(value);
   return <YearLabel dx="2">{formatted}</YearLabel>;
 };
 
-const prepLineChartData = (chartData) => Object.values(chartData.filter((entity) => entity.isGroup).reduce((memo, entity) => {
-  const { group } = entity;
-  const updatedMemo = memo;
-  if (!updatedMemo[group]) {
-    updatedMemo[group] = [];
-  }
-  return { ...updatedMemo, [group]: [...updatedMemo[group], entity] };
-}, {}));
+const getEntityCodeById = (entities, id) => entities.find((entity) => qe(entity.get('id'), id)).getIn(['attributes', 'code']);
+
+const prepLineChartData = (chartData) => Object.values(chartData.filter((entity) => entity.isGroup)
```

Comment:
@vbojilova surely this could be improved w/out the `Object.values` and `updatedMemo`